docs/usage_guide.rst

```diff
@@ -701,6 +708,15 @@ An xpub does not allow spending your coins but provides information about your w
    :align: center
 
 
+ENS names resolution
+=====================
+
+rotki automatically resolves ens name for each your ethereum account. If there is an ens name for an account, this name will be shown instead of raw blockchain address. You can find the blockchain address by hovering the ens name.
```

Comment:
Isn't it better?
```suggestion
rotki automatically resolves ens name for each of your ethereum accounts. If there is an ens name for an account, this name will be shown instead of raw blockchain address across the app. You can find the blockchain address by hovering the mouse over the ens name.
```

Reply:
Sounds cool, will change it, thks

Reply:
Not exactly. It's all cool but please specify: 

`If there is an ens name for an account` -> `If there is a primary ens name specified for an account`

This is because if you don't set the primary ens name for your address it won't be shown. 

P.S.: Do you have an ENS @nebolax? you should get one!

Reply:
Oh yeah, thank you Lefteris!